gammapy/irf/exposure.py

```diff
@@ -0,0 +1,54 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+import numpy as np
+from ..data import SpectralCube
+from astropy.coordinates import SkyCoord, Angle
+
+import logging
+
+__all__ = [
+    'exposure_cube'
+]
+
+log = logging.getLogger(__name__)
```

Comment:
remove `import logging` and this line ... it's only needed from files where we do want to log, which is rare and I don't think needs to be added "just in case we want to emit log messages later"